estimagic/optimization/scipy_optimizers.py

```diff
@@ -0,0 +1,137 @@
+import functools
+
+import numpy as np
+import scipy
+
+from estimagic.optimization.default_algo_options import GRADIENT_TOLERANCE
+from estimagic.optimization.default_algo_options import LIMITED_MEMORY_STORAGE_LENGTH
+from estimagic.optimization.default_algo_options import MAX_CRITERION_EVALUATIONS
+from estimagic.optimization.default_algo_options import MAX_ITERATIONS
+from estimagic.optimization.default_algo_options import MAX_LINE_SEARCH_STEPS
+from estimagic.optimization.default_algo_options import RELATIVE_CRITERION_TOLERANCE
+
+
+DEFAULT_ALGO_INFO = {
+    "primary_criterion_entry": "value",
+    "parallelizes": False,
+    "needs_scaling": False,
+}
+
+
+def scipy_lbfgsb(
+    criterion_and_derivative,
+    x,
+    lower_bounds=None,
+    upper_bounds=None,
```

Comment:
No defaults? https://estimagic--163.org.readthedocs.build/en/163/explanations/optimization/internal_optimizers.html#the-internal-optimizer-interface

```suggestion
    lower_bounds,
    upper_bounds,
```

Reply:
Will fix this manually because it occurs at many places